src/nettests/ndt/index.js

```diff
@@ -0,0 +1,101 @@
+exports.formatRunSummary = (summary, {React, Cli, Components, chalk}) => {
```

Comment:
`summary` is the actual summary object, `React` is actually React.js when you are on desktop (i.e. `React = require('react')`), `Cli` is some OONI specific command line interface proxy object, `Components` is the result of `require('ooni-components')` (question to self, do we want to include cli components in `ooni-components`?), `chalk` is the `chalk` library for making pretty CLI stuff.